test/Testing/LeanCode.IntegrationTestHelpers.Tests/ConfigurationOverridesTests.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace LeanCode.IntegrationTestHelpers.Tests;
+
+public class ConfigurationOverridesTests
+{
+    [Fact]
+    public void TestBuild()
```

Comment:
This should be split into a number of tests:
1) Test if empty overrides results in empty configuration (this is the most basic case),
2) Test if a single `AddValue` works,
3) Test if multiple (separate) `AddValue` works,
4) Test if `AddValue` called multiple times (2) with the same key work as expected,
5) The same set of 2-4 tests for connection strings,
6) And a mix of both `AddValue` and `AddConnectionString` (but a simple test with a single call to both).